model/annotations/task_annotations_test.go

```diff
@@ -158,3 +158,21 @@ func TestMoveSuspectedIssueToIssue(t *testing.T) {
 	assert.Equal(t, "different user", annotationFromDB.Issues[0].Source.Author)
 	assert.Equal(t, "someone new", annotationFromDB.Issues[1].Source.Author)
 }
+
+func TestValidateIssueURL(t *testing.T) {
+	issue := IssueLink{URL: "issuelink.com"}
+	err := ValidateIssueURL(issue.URL)
+	assert.Contains(t, err.Error(), "error parsing request uri")
+
+	issue.URL = "http://issuelink.com/"
```

Comment:
Does this also work without the dash at the end, with just  "http://issuelink.com"?

Reply:
Yup, it does! `/` is counted as part of the path, so the host `issuelink.com` is still valid.